chart/templates/21-zeek-live.yml

```diff
@@ -79,6 +79,8 @@ spec:
               name: upload-common-env
           - configMapRef:
               name: zeek-env
+          - configMapRef:
```

Comment:
would not include this unless kafka is enabled.

Reply:
Done.